ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -26,7 +26,8 @@ class UnityEnvironment(object):
 
     def __init__(self, file_name=None, worker_id=0,
                  base_port=5005, seed=0,
-                 docker_training=False, no_graphics=False):
+                 docker_training=False, no_graphics=False,
+                 timeout_wait=30):
```

Comment:
personally I'd call this something like `communicator_timeout` or `connection_timeout` -- no big deal if you think this is easier to understand though.